tests/Rule/ContainsNumericCharactersTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Tests\Morebec\Validator\Rule;
+use InvalidArgumentException;
+use Morebec\Validator\Rule\ContainsNumericCharacters;
+use PHPUnit\Framework\TestCase;
+
+class ContainsNumericCharactersTest extends TestCase
+{
+    public function testValidate(){
+        $ruleFirst = new ContainsNumericCharacters(1,true);
+        $ruleSecond = new ContainsNumericCharacters(1,false);
+        $ruleThird= new ContainsNumericCharacters(1,false,"Custom Message");
```

Comment:
Testing with various values instead of only '1' would be better for test coverage.